ledger_secure_sdk_sys/build.rs

```diff
@@ -477,7 +477,7 @@ impl SDKBuilder {
         );
 
         let mut bindings = bindgen::builder()
-            .clang_args(&args)
```

Comment:
Do we need this change to be included in ledger-device-sdk crate?

Reply:
I'm afraid of apps using the last sys crate version in their cargo.toml as this end with building twice the crate with two different version, and generates an invalid app binary.

Reply:
ok let's postpone this modification for later then...